### ISSUE TITLE
Deleted Words from save.json Automatically Recreated

### ISSUE THREAD

**Issue by Matt-662607015** [IlyaChichkov/LethalAutocompleteMod#9]
I spent a long time editing save.json by adding, renaming, and deleting words. After I was done editing I opened the game, loaded a new save, opened the terminal, and closed the terminal. As soon as the terminal closed, a massive number of unwanted words that I had renamed/deleted were automatically saved to the bottom of save.json. I'll rename and upload the two versions of save.json as before.json and after.json for your reference. I'll set the weights for the unwanted words to 0 as a temporary solution, but they're really clogging up both the file and terminal, so I'm hoping you can find a way to stop deleted words from reappearing.

[before.json](https://github.com/IlyaChichkov/LethalAutocompleteMod/files/13982677/before.json)
[after.json](https://github.com/IlyaChichkov/LethalAutocompleteMod/files/13982679/after.json)

**Comment by IlyaChichkov** [comment]
Hello, Matt, I've just uploaded a new version [v0.4.2](https://github.com/IlyaChichkov/LethalAutocompleteMod/tree/0.4.2) with history and autocomplete commands blacklists that you can customize in the same save.json file. I hope this should solve the problem you have. However, if you find that the problem persists or if you encounter any other issues, please don't hesitate to reach out and let me know!

**Comment by Matt-662607015** [comment]
I appreciate you working on this issue. Unfortunately the commands blacklist doesn't seem to be working for me. I cleared my mod cache and reinstalled, but I still don't see the commands blacklist in save.json. I tried adding it myself in two different ways, but both are deleted whenever I leave the terminal. I'll upload pictures showing the two ways I tried.

![CommandsBlacklist1](https://github.com/IlyaChichkov/LethalAutocompleteMod/assets/41337151/574c4bd2-39d1-41cd-a736-4bf199b7b27e)
![CommandsBlacklist2](https://github.com/IlyaChichkov/LethalAutocompleteMod/assets/41337151/5f2d6818-4b31-4e4c-9dd8-60fedbb89f18)
